vitamins/io.github.spacingbat3.webcord.metainfo.xml

```diff
@@ -44,6 +44,7 @@
   </content_rating>
 
   <releases>
+    <release version="4.3.0" date="2023-06-23" urgency="medium" />
```

Comment:
Please populate the release with metainfo.

Reply:
They added it later. When I opened the PR there were no release notes.